aaad/aaad.py

```diff
@@ -54,6 +58,14 @@ def do_GET(self):
 
         content_type = self.headers.getheader("content-type", "")
 
+        if auth_action == "filter_response":
+            filtered_response = self.filter_response(body, resource, act_as_user, permissions)
+            self.send_response(200)
```

Comment:
what if the response is not 200 (201 e.g.)?
what if the response is a not 2xx (say 4xx or 5xx etc.)?